common/models.py

```diff
@@ -181,7 +181,9 @@ class Meta:
 class BaseModelMandatoryName(BaseModel):
     """Changed BaseModel.name to blank=False."""
 
-    name = models.CharField(max_length=128, blank=False, default="", null=False)
+    name = models.CharField(
+        max_length=128, blank=False, default="", null=False, unique=True
```

Comment:
As discussed, the issue is whether something should be _globally_ unique or just unique per institution (as with Experiment). But I agree that we just defer this to another issue because it's speculative as to whether it would even be a problem. See #225